### ISSUE TITLE
PDF "continued on next page" for long tables isn't internationalized

### ISSUE THREAD

**Issue by jfbu** [sphinx-doc/sphinx#3791]
### Problem
when a table is split across pages, there is  a ``continued on next page`` and a ``continued from previous page``. They are inserted via the longtable template. Although the template has the mark-up for translated strings, the output contains only the English version, independent of config value language.

#### Procedure to reproduce the problem
```
.. csv-table:: A long table
   :class: longtable

   a, b, c
   a, b, c
   a, b, c
   a, b, c
   a, b, c
   ... many such lines,,
   a, b, c
   a, b, c
   a, b, c
   a, b, c

```
and for example ``language = 'it'`` in `conf.py`.

#### Error logs / results
![capture d ecran 2017-05-23 a 15 31 19](https://cloud.githubusercontent.com/assets/2589111/26356632/fa21cd4e-3fcc-11e7-83d2-e45192052e59.png)

### Environment info
- OS: Mac
- Python version: 2.7.13 (CPython)
- Sphinx version: 1.6.1
- Jinja2 version: 2.9.6

**Comment by jfbu** [comment]
It appears that the `.po` files in sphinx/locale/<language> simply did not extract strings from the templates. I am working on an issue #3764 where I would need to insert localizable strings in ``latex.tex_t`` also.

**Comment by tk0miya** [comment]
Fixed at 6324946.

Now it become translatable again, but we lost translation text because it is dropped from translation targets for a while...
